packages/pds/src/config/config.ts

```diff
@@ -55,6 +55,15 @@ export const envToCfg = (env: ServerEnvironment): ServerConfig => {
   }
   if (env.blobstoreS3Bucket) {
     blobstoreCfg = { provider: 's3', bucket: env.blobstoreS3Bucket }
+    if (env.blobstoreS3Region) {
+      blobstoreCfg.region = env.blobstoreS3Region
+    }
+    if (env.blobstoreS3Endpoint) {
+      blobstoreCfg.endpoint = env.blobstoreS3Endpoint
+    }
+    if (env.blobstoreS3ForcePathStyle !== void 0) {
```

Comment:
any reason for using `void(0)` here instead of `undefined`?

Reply:
That's a kind of my habit because `undefined` was once writable and even nowadays it can be shadowed.
(Ref.: https://eslint.org/docs/latest/rules/no-undefined)

Alternative is

```typescript
    if (tyepof(env.blobstoreS3ForcePathStyle) == 'boolean') {
```